utils/remove_longshortdata.sh

```diff
@@ -32,12 +32,12 @@ cat ${sdir}/utt2num_frames \
 echo "remove utterances having more than $maxchars or less than $minchars characters"
 # counting number of chars
 if [ -z ${nlsyms} ]; then
-text2token.py -s 1 -n 1 $sdir/text \
+tutils_espnet/ext2token.py -s 1 -n 1 $sdir/text \
```

Comment:
tutils_espnet
->
utils_espnet

Reply:
Actually I think we may not need `utils_espnet/` anymore, right? Can you fix this?